DEPRECATED - DO NOT MERGE W-13582136 ACB structure hn (#55)
mulesoft/docs-code-builder: modules/ROOT/pages/tutorial-design-api-spec.adoc

= Design an API Specification

include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"] 

Anypoint Code Builder enables you to design new RAML or OAS API specifications.

Supported OAS and RAML versions are:

* RAML 1.0
* RAML 0.8
* OAS 2.0 (JSON)
* OAS 2.0 (YAML)
* OAS 3.0 (JSON)
* OAS 3.0 (YAML)

> The following section teaches you how to design your first API specification, reuse API fragments from Exchange, test your API using API Console, save your API to Design Center, and finally publish the API as an asset to Exchange.

Reviewer: ```suggestion
The following sections are intended to help you design your first API specification, reuse API fragments from Exchange, test your API using API Console, save your API to Design Center, and finally publish the API as an asset to Exchange.
```
